src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/Prison.kt

```diff
@@ -35,4 +35,20 @@ data class Prison(
   @Temporal(TemporalType.TIMESTAMP)
   @Column
   val modifyTimestamp: LocalDateTime? = null
-)
+) {
+  override fun equals(other: Any?): Boolean {
```

Comment:
@AledWEvans - Haven't added the parent class for entities as you suggested yesterday. Could we pick that up as part of  different code improvements change?

Reply:
sure